client/reader/stream/post.jsx

```diff
@@ -171,6 +171,13 @@ export default class ReaderPostCardAdapterFluxContainer extends React.Component
 		FeedPostStore.off( 'change', this.updateState );
 	}
 
+	shouldComponentUpdate( nextProps, nextState ) {
+		const shouldUpdate = this.props !== nextProps ||
```

Comment:
ah, that could be the cause of some leftover wasted renders! thanks for explaining, i'll toss that in